package.json

```diff
@@ -18,6 +18,7 @@
 	"devDependencies": {
 		"@istanbuljs/nyc-config-typescript": "^1.0.1",
 		"@types/chai": "^4.2.21",
+		"@types/glob": "^7",
```

Comment:
Any concerns with letting the types version here float apart from the version number?

Reply:
I don't think so, we can have a fixed minor version if you want, but if both are using the same major version it won't be a problem.
What do you think @arielmelendez ?

Reply:
Should be fine as long as it's using semantic versioning adequately 🤞